lib/sidekick.rb

```diff
@@ -51,20 +51,27 @@ def verbose
     require 'json'
 
     # Generates a URL for the request
-    def url_for_request(query)
-      "https://api.github.com/search/issues?q=#{URI.escape query}&repo=#{repo_owner}/#{repo_name}&sort=created&order=asc"
+    def url_for_request(query, sort_by: nil, order: nil)
```

Comment:
When I compared crashes to search results, the sort_by and order were pretty important in deciding, especially when there was a few similar results